Mathlib/CategoryTheory/Bicategory/Functor/Strict.lean

```diff
@@ -0,0 +1,56 @@
+/-
```

Comment:
It seems this will conflict with the already existing file `Bicategory.Functor.Strict`.